engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbJobState.java

```diff
@@ -376,11 +391,11 @@ private void makeJobNotActivatable(final DirectBuffer type) {
     EnsureUtil.ensureNotNullOrEmpty("type", type);
 
     jobTypeKey.wrapBuffer(type);
-    activatableColumnFamily.delete(typeJobKey);
+    activatableColumnFamily.deleteIfExists(typeJobKey);
   }
 
   private void removeJobDeadline(final long deadline) {
     deadlineKey.wrapLong(deadline);
-    deadlinesColumnFamily.delete(deadlineJobKey);
+    deadlinesColumnFamily.deleteIfExists(deadlineJobKey);
```

Comment:
:x: Pretty sure this should be a _deleteExisting_. Please try run tests, etc.

Reply:
That doesn't work at all, lot's of failing tests. There doesn't seem to be a specific test for this but `shouldFailJobAndRetryWithBackOff` is one that fails.